docs/general/clients/codec-support.md

```diff
@@ -65,6 +65,24 @@ The goal is to Direct Play all media. This means the container, video, audio and
 <br />
 <sup>2</sup><a href="https://en.wikipedia.org/wiki/MPEG-4_Part_17">MPEG-4 Part 17: MP4TT Subtitles</a>
 
+### HDR Support
+
+HDR is only supported on a very limited range of devices:
+
+- iOS Devices with an HDR capable internal display can display HDR content when using a browser or the Jellyfin iOS App (not Swiftfin) as long as the codec is supported by the client. All of them support HEVC and VP9, while AV1 is supported on newer devices. HDR10, Dolby Vision and HLG are supported.
+
+- Android devices running either the official Android App or using Chrome can display HDR content if the device supports HDR and the source format. Format support depends on device.
+
+- Macs running macOS that supports HDR can play HDR content when using Safari, Chrome or Firefox on an HDR capable display as long as the source codec is supported. Safari has the most complete support, including HDR10, Dolby Vision (including P5) and HLG. Chrome supports HDR10 and Dolby Vision (except P5), while Firefox only supports HDR10 on VP9 and AV1 content. Please refer to [the list of Macs that support HDR](https://support.apple.com/en-us/102205) to check if your device is supported.
+
+- Windows PCs with an HDR display with HDR enabled in the settings using Edge or Chrome. Dolby Vision requires the Dolby Vision addon from the Microsoft Store to be installed.
```

Comment:
DV extension only works in edge and edge has generally better HDR support on windows, like having a better client-side tone mapping implementation. A lot of users have reported that edge produces much better tone mapped playback than chrome.